### PR TITLE
L5TEAM-36 <merge to remote the last update prorotype>

### PR DESCRIPTION
**Pull request by buchelij**

### CODE REVIEW

**Review by smitpatel64 — COMMENTED** [review]
housingapp

**Review by smitpatel64 — CHANGES_REQUESTED** [review]
housingapp

File: main/templates/main/aboutPage.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{% extends 'main/base.html' %}
```
Comment (buchelij) [inline main/templates/main/aboutPage.html:1]: Kamal contribution

**Review by buchelij — COMMENTED** [review]
adasd

**Review by buchelij — COMMENTED** [review]
good

**Pull request merged**